BenchPress/Generators/AzureDeploymentImporter.cs

```diff
@@ -5,12 +5,21 @@ namespace Generators;
 
 public class AzureDeploymentImporter
 {
+    private static Regex s_parametersOrVariablesRegex = new Regex(
+        "(?<paramOrVarType>parameters)\\(\\'(?<paramOrVarName>.*?)\\'\\)|(?<paramOrVarType>variables)\\(\\'(?<paramOrVarName>.*?)\\'\\)",

```

Comment:
I found [some documentation](https://learn.microsoft.com/en-us/dotnet/standard/base-types/grouping-constructs-in-regular-expressions#:~:text=Note%20that%20a%20group%20name%20can%20be%20repeated%20in%20a%20regular%20expression.) about grouping constructs and reusing group names. It also worked with my testing. I do admit that my regex is super rusty, so I may be interpreting the docs incorrectly! 

That aside, you're definitely right that this regex can probably be simplified! Will try out your suggestions.

Reply:
Interesting. I figured there was something that I was missing. It might be Regex flavor thing. I was using https://regex101.com/ and it didn't like having the group name repeated.